### ISSUE TITLE
stylistic set for letter t to have same base as r, l, i, f

### ISSUE THREAD

**Issue by enriquemartinez-emc** [githubnext/monaspace#223]
Hi, first of all thank you so much for developing these beautiful fonts!

I'd like to request a stylistic set for the letter t to have the same base as the letters mentioned above. 

The letter t has this umbrella-like shape which IMO is a bit distracting since the rest of the other letters have a different base.

Pd: Sorry, I forgot to mention specifically Monaspace Neon :-D

Thanks again!

**Comment by gmosx** [comment]
Indeed, t looks a bit weird, I guess I can get used to it, but it would be nice to have another variant!

**Comment by idan** [comment]
Fixed in 1.2, use `cv11`

**Comment by enriquemartinez-emc** [comment]
Thank you so much!
